src/content/language/type-system.md

```diff
@@ -259,7 +259,7 @@ because it's an error to cast a list of dogs to a list of cats:
 ```dart tag=runtime-fail
 void main() {
   List<Animal> animals = [Dog()];
-  List<Cat> cats = [!animals as List<Cat>!];
+  List<Cat> cats = [!animals as List<Dog>!];
```

Comment:
> It seems likely that it was written under the assumption that animals would have a run-time type that implements List<Dog>. That's not true, however, because the context type List<Animal> is given priority, and hence [Dog()] means <Animal>[Dog()].

Yes, now I understood. thank you.

Reply:
Thanks!

Reply:
Do you wish to change the lines I mentioned, to something like this?:

```dart
List<Animal> animals = <Dog>[Dog()];
List<Cat> cats = [!animals as List<Cat>!];
```

I think that would actually clarify the information provided by this page.

Reply:
Sure, I will. :)